assets/src/components/v2/departures/headway_section.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import FreeText from "Components/v2/dup/dup_free_text";
+
+const HeadwaySection = ({ text, isOnlySection }) => {
+  const className = isOnlySection ? "full-screen-headway" : "partial-alert";
```

Comment:
`isOnlySection`: Would it be possible to determine this property directly in the CSS using an `:only-child` pseudo-class, instead of communicating it from JS to CSS via a classname? (If so, would that be an improvement?)

Reply:
Yeah, I think so. Good call.